athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/ListTablesRequest.java

```diff
@@ -31,24 +31,35 @@
 public class ListTablesRequest
         extends MetadataRequest
 {
+    public static final int NULL_PAGE_SIZE = -1;
```

Comment:
I dont think this name properly conveys what this value indicates. Also, every public member should have javadoc.
Can we rename this INF_PAGE_SIZE_VALUE or similar? -1 basically means you shouldn't limit the size of the pages.

Reply:
Added javadoc and renamed to `UNLIMITED_PAGE_SIZE_VALUE`.